tests/acceptance/test-helper.ts

```diff
@@ -1,21 +1,30 @@
-import { PocketGatewayApplication } from '../..'
+import { PocketGatewayApplication } from '../../src/application'
 import { createRestAppClient, givenHttpServerConfig, Client } from '@loopback/testlab'
+import { gatewayTestDB } from '../fixtures/test.datasource'
+import RedisMock from 'ioredis-mock'
+import rewiremock from 'rewiremock'
```

Comment:
While this is super nitpicky, I feel over time this is important for code quality. Could we order the imports affected by this PR in a certain way? I'd go with:
- Library imports (`ioredis-mock`, etc) without `@` first
- Library imports with `@` after (loopback)
- Then relative imports.
  - I also like to alphabetize things, so I usually alphabetically order these as well.

And, I really like to alphabetize things, so, while I think you've actually been doing, we could also alphabetize the actual imports if they're destructured, like in line 2. :smile:

Reply:
And thinking about this, we could probably invert the two first rules to have loopback specific imports first, as this is a loopback project, just as its common to have express imports on top or React imports, for that matter.

Reply:
I do certainly agree with that, we can also add this to the `CONTRIBUTING.md` file

Reply:
Isn't this something linting can do automatically?

Reply:
yep, I don't think at the specific level @Evalir mentioned but at least one good enough, on it

Reply:
Yeah, this can probably be achieved to some degree with linting, but there'll still be some nitpicking to do afterward haha.